### PR TITLE
Add more tests for Azure cloud provider.

### PR DESCRIPTION
**Pull request by brendandburns**
<eom>

### CODE REVIEW

File: pkg/cloudprovider/providers/azure/azure_storage.go [hunk 1]
```diff
@@ -24,51 +24,48 @@ import (
...
+		accounts, errResult = az.getStorageAccounts()
+		if errResult != nil {
 			// TODO: create a storage account and container
-			return "", "", err
+			return "", "", errResult
 		}
 	}
 	for _, account := range accounts {
-		glog.V(4).Infof("account %s type %s location %s", account.Name, account.StorageType, account.Location)
+		glog.Errorf("account %s type %s location %s", account.Name, account.StorageType, account.Location)
```
Comment (andyzhangx) [inline pkg/cloudprovider/providers/azure/azure_storage.go]: keep it as `glog.V(4).Infof`, it's not an error

**Review by andyzhangx — CHANGES_REQUESTED** [review]

File: pkg/cloudprovider/providers/azure/azure_storage.go (hunk 1 above)
Comment (brendandburns) [reply pkg/cloudprovider/providers/azure/azure_storage.go]: Good catch, added that for debugging, reverted.

**Comment by brendandburns** [comment]
Comment addressed, ptal.

Thanks!

**Review by andyzhangx — APPROVED** [review]

**Pull request merged**